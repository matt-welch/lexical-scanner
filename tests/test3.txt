12.3e+10
45.2E10
4.269E97
98.7e-10
32.1E+11
7.65e-13
1.23e12
4.56e+12
789e+12
func_name
FUNC
123

