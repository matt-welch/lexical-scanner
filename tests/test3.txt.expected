1 12.3e+10 
2 45.2 
2 E10 
3 4.269 
3 E97 
4 98.7e-10 
5 32.1E+11 
6 7.65e-13 
7 1.23
7 e12
8 4.56e+12
9 789
9 e
9 +
9 12
10 func
10 _
10 name
11 FUNC
12 123

